catalog/core/catalog-core-urlresourcereader/src/main/java/ddf/catalog/resource/impl/URLResourceReader.java

```diff
@@ -77,23 +80,16 @@
 
     private static final String DEFAULT_MIME_TYPE = "application/octet-stream";
 
-    private static final int QUALIFIER_SET_SIZE = 3;
-
     private static final String BYTES_TO_SKIP = "BytesToSkip";
-
-    private static Set<String> qualifierSet;
-
-    static {
-        qualifierSet = new HashSet<String>(QUALIFIER_SET_SIZE);
-        qualifierSet.add(URL_HTTP_SCHEME);
-        qualifierSet.add(URL_HTTPS_SCHEME);
-        qualifierSet.add(URL_FILE_SCHEME);
-    }
+    
+    private final Set<String> qualifierSet = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(new String[] {URL_HTTP_SCHEME, URL_HTTPS_SCHEME, URL_FILE_SCHEME})));
```

Comment:
You don't need to make a string array here. Aslist takes individual object args as well.